chain/block.go

```diff
@@ -298,12 +253,12 @@ func (b *StatelessBlock) initializeBuilt(
 func (b *StatelessBlock) ID() ids.ID { return b.id }
 
 // implements "block.WithVerifyContext"
-func (b *StatelessBlock) ShouldVerifyWithContext(context.Context) (bool, error) {
-	return b.containsWarp, nil
+func (*StatelessBlock) ShouldVerifyWithContext(context.Context) (bool, error) {
+	return false, nil
 }
 
 // implements "block.WithVerifyContext"
-func (b *StatelessBlock) VerifyWithContext(ctx context.Context, bctx *block.Context) error {
+func (b *StatelessBlock) VerifyWithContext(ctx context.Context, _ *block.Context) error {
```

Comment:
I think we can remove the usage of:
* `BuildBlockWithContext`
* `ShouldVerifyWithContext`
* `VerifyWithContext`